docker/teleport-quickstart.yml

```
@@ -3,7 +3,7 @@ services:
   # The configure container starts, generates a config, writes it to
   # /etc/teleport/teleport.yaml and then immediately exits.
   configure:
-    image: quay.io/gravitational/teleport:5
+    image: quay.io/gravitational/teleport:6.1.5
```

The old `5` tag was a 'nightly' tag - i.e it is rebuilt every night and points to the latest `5.x` Teleport version. We should use the equivalent `6` tag here.

[Here's a list of tags](https://quay.io/repository/gravitational/teleport?tag=latest&tab=tags) - see the `4.4`, `5` and `6` at the top.

```suggestion
    image: quay.io/gravitational/teleport:6
```